index.html

```diff
@@ -950,11 +956,257 @@ <h3>
         <h3>
           Unnecessary Requests for Credentials
         </h3>
-        <p class="issue" title="Work in progress">
-          Explain how the API could be used to unnecessarily request digital
-          credentials from individuals such as requesting a driver's license to
-          log into a movie rating website and how the ecosystem can mitigate
-          this risk.
+        <p class="issue" title="Work in progress"></p>
+        <p>
+          Unnecessary credential requests are a key privacy risk of the entire
+          digital credentials ecosystem. They could manifest in different ways
+          and from different motivations:
+        </p>
+        <ul>
+          <li>Intentional abuse of the API to learn sensitive information about
+          the user for the purpose of fraud, tracking, or sale of the data. For
+          example, a site could trick a user into sharing their passport
+          information through misleading content. This can lead to identity
+          theft and financial loss, and severe loss of control and/or leakage
+          of personal information.
+          </li>
+          <li>Unnecessary requests for credentials without the explicit intent
+          of user harm, such as an online store requesting users to sign up
+          with their driver's license instead of generic email & password or
+          federated credentials. This can lead to exclusion of users without
+          the ability or willingness to share such a credential with the site,
+          a deterioration of the prompt experience on the web, and an increase
+          in the risk of accidental data leakage.
+          </li>
+          <li>Requests for an excessive amount of information for valid
```

Comment:
Again, Issuing Authorities should limit what should be requested - but also RPs and protocols should restrict oversharing.

Reply:
Same here, closing this out but please lmk if you were expecting a follow up here.